python-frontend/src/main/java/org/sonar/plugins/python/api/tree/ComprehensionExpression.java

```diff
@@ -19,10 +19,15 @@
  */
 package org.sonar.plugins.python.api.tree;
 
+import java.util.Set;
+import org.sonar.python.semantic.Symbol;
+
 public interface ComprehensionExpression extends Expression {
 
   Expression resultExpression();
 
   ComprehensionFor comprehensionFor();
 
+  Set<Symbol> localVariables();
```

Comment:
I think it's worth mentioning in a comment we chose to implement Python 3 scoping rules for Comprehension